app/views/users/_all_users_list.html.erb

```diff
@@ -0,0 +1,28 @@
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Email</th>
+    <th>Role</th>
+    <th>Status</th>
+    <th></th>
+    <th></th>
+  </tr>
+  <% @users.each do |user| %>
```

Comment:
@apeksha2697  Please use rails collection and partial on view,
read about this https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections
and implement it in the similar way